### PR TITLE
Prepare for registry publishing

### PR DESCRIPTION
**Pull request by CPatchane**
Waiting for the new registry token

### CODE REVIEW

**Comment by CPatchane** [comment]
[(3) ETDéveloppeur j'ai publié un connecteur](https://trello.com/c/5mMc9bRq/857-3-etd%C3%A9veloppeur-jai-publi%C3%A9-un-connecteur)

File: manifest.konnector [hunk 1]
```diff
@@ -1,33 +1,29 @@
 {
-  "version": "3.0.0",
+  "version": "0.1.0",
   "name": "Trainline",
   "type": "node",
```
Comment (CPatchane) [inline manifest.konnector]: We have to change that type to `konnector` when our backend will be able to handle the `language: node` instead.

File: manifest.konnector [hunk 2]
```diff
@@ -1,33 +1,29 @@
...
-  "slug": "trainline",
-  "source": "git://github.com/cozy/cozy-konnector-trainline.git",
+  "slug": "konnector-trainline",
+  "source": "git://github.com/cozy/cozy-konnector-trainline.git#build",
+  "editor": "Cozy",
+  "vendorLink": "www.trainline.fr",
```
Comment (gregorylegarec) [inline manifest.konnector]: `vendorLink` becomes `vendor_link` in manifest v2.

File: manifest.konnector [hunk 3]
```diff
@@ -1,33 +1,29 @@
...
+      "type": "password"
+    },
+    "advancedFields": {
+      "folderPath": {
+        "advanced": true,
+        "isRequired": false
       }
     }
   },
+  "dataType": [
```
Comment (gregorylegarec) [inline manifest.konnector]: ⚠️ `dataType` becomes `data_type` in manifest v2.

**Comment by gregorylegarec** [comment]
Thanks @CPatchane 💊

**Pull request merged**